### DISCUSSION TITLE
v1.54 changelog?

### DISCUSSION THREAD

**Discussion by raburton** [phil-barrett/PicoCNC#27] [General]
Just wondered what the differences are with v1.54, as there isn't any mention of it on github.

**Comment by phil-barrett** [comment]
Sorry for the late response, I was on a boat in the Aegean.

1.54 adds a 5 volt header in the upper right corner.  Useful for extra 5v power.

**Comment by raburton** [comment]
No problem. The Aegean is lovely, I was swimming in it earlier this year.

Do you mean the 5v Aux power pin header, next to the a-axis pins? That's shown on the picture of the 1.53 board too.

**Comment by phil-barrett** [comment]
Hmmm.  I'll go back and look.  There are no operational differences for the firmware.
